python/pynq/intf/__init__.py

```diff
@@ -39,6 +39,7 @@
 from .trace_analyzer import TraceAnalyzer
 from .fsm_generator import FSMGenerator
 
+
```

Comment:
I changed the modes of those files to R+W+E. I thought people would want to run it directly so changing the modes makes it easier.